src/data/apischema.py

```diff
@@ -42,9 +44,14 @@ class GetRecommendationFilter(BaseModel):
     task_id: Optional[int] = None
     date: Optional[str] = None
     location: Optional[str] = None
-    severity: Optional[str] = None
```

Comment:
why not make it a  dict with min, max attribute ?

Reply:
true. changed it